json-resume-builder/src/components/Work.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+import resumeData from "../resume.json";
+
+export default class Work extends Component {
+  render() {
+    const work = resumeData.work;
+    return (
+      <section>
+        {work.map(work => {
+          return (
+            <article>
+              <h2>{work.company}</h2>
+              <h3>{work.position}</h3>
+              <h4>
+                {work.startDate} to {work.endDate}
+              </h4>
+              <ul>
+                {work.highlights.map(summary => {
+                  return <li>{summary}</li>;
+                })}
```

Comment:
can do `{work.highlights.map(summary => <li>{summary}</li>)}`
Also, you should be adding `key` to these